src/shogun/transfer/multitask/MultitaskROCEvaluation.cpp

```diff
@@ -68,7 +71,7 @@ float64_t CMultitaskROCEvaluation::evaluate(CLabels* predicted, CLabels* ground_
 		//m_tasks_indices[t].display_vector();
 		predicted->add_subset(m_tasks_indices[t]);
 		ground_truth->add_subset(m_tasks_indices[t]);
-		result += evaluate_roc(predicted,ground_truth)/m_tasks_indices[t].vlen;
+		result += evaluate_roc((CBinaryLabels*)predicted,(CBinaryLabels*)ground_truth)/m_tasks_indices[t].vlen;
```

Comment:
if you do static casts, be more explicit, i.e. put a comment that this is ok, or do the cast just below the assertions above and store the result in a variable